go.mod

```diff
@@ -83,7 +83,7 @@ require (
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.46.1
 	go.opentelemetry.io/otel v1.21.0
 	go.opentelemetry.io/otel/exporters/jaeger v1.17.0
-	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.42.0
+	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc v0.44.0
 	go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetrichttp v0.42.0
```

Comment:
This was updated to v0.44.0 in https://github.com/docker/cli/commit/89db01ef977cdd868dddde4c211e58f74457bacb#diff-3cf15fd4671254ae18dff1505434826e26a59cdc6e0d945dd4da1e0d63819602 - should these (and other v0.42.0 versions) stay aligned,  and all be v0.44.0 @jsternberg ?